sherpa/astro/data.py

```diff
@@ -4645,7 +4700,10 @@ def notice_response(self, notice_resp=True, noticed_chans=None):
             arf, rmf = self.get_response(resp_id)
             _notice_resp(noticed_chans, arf, rmf)
 
-    def notice(self, lo=None, hi=None, ignore=False, bkg_id=None):
+    def notice(self, lo=None, hi=None,
```

Comment:
Is there a reason that there is no type hint for "lo" and "hi"? 
If we type a function, I would prefer to type all parameters (except in special cases where it's just too hard to do)